src/style.css

```diff
@@ -401,7 +399,7 @@ div.youtube-playlist li {
   border-left: 4px solid #cc4141;
   border-radius: 0 .25rem .25rem 0;
   margin: 1rem 0;
-  color: #5b5656;
+  color: #333;
```

Comment:
See my other comment - can we move this to Tailwind config and reference it here?

Reply:
Where do we check the Tailwind config for colors?

I am OK using them, but not sure where these are set.

1. Can you do a "Search and replace" for all `#333` and `#333333` and `#000` and `#000000` ?

Reply:
> Where do we check the Tailwind config for colors?

@tiagov8

https://github.com/SSWConsulting/SSW.Rules/blob/ad38e10997c5fa337af859b3c1443b7cb39d903e/tailwind.config.js#L17-L23